src/org/javarosa/xml/util/UnfullfilledRequirementsException.java

```diff
@@ -0,0 +1,82 @@
+package org.javarosa.xml.util;
+
+/**
+ * @author ctsims
+ */
+public class UnfullfilledRequirementsException extends Exception {
```

Comment:
This exception doesn't seem to be thrown anywhere. Only caught in `ExternalDataInstance`'s contructor and declared in `ElementParser#parse()` method signature.